include/staff/role.inc.php

```diff
@@ -80,6 +80,7 @@
                 if ($v['primary'])
                     continue;
                     $buckets[$g][$k] = $v;
+                asort($buckets[$g]);
```

Comment:
I don't think you mean to put this here? It's inside the `foreach` for `$k`, so you'll be sorting it for each key added to `$buckets[$g]`